java/ql/src/Security/CWE/CWE-079/AndroidWebViewAddJavascriptInterface.ql

```diff
@@ -0,0 +1,17 @@
+/**
+ * @id java/android-webview-addjavascriptinterface
+ * @description Exposing a Javascript interface to a Java object in a WebView can lead to malicious JavaScript controlling the application.
+ * @kind problem
+ * @problem.severity warning
+ * @security-severity 6.1
+ * @precision high
```

Comment:
Since this won't always be a security problem, let's get the precision down to `medium`. That way this query won't run by default, i.e. it will be opt-in for users that want to see more results.